src/main/java/com/simanavets/tenderflexbackend/controller/UserController.java

```diff
@@ -0,0 +1,30 @@
+package com.simanavets.tenderflexbackend.controller;
+
+import com.simanavets.tenderflexbackend.entity.User;
+import com.simanavets.tenderflexbackend.service.UserService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/users")
+public class UserController {
+
+    private final UserService userService;
```

Comment:
We must use more flexible connections between objects, so it's very bad practice to inject Object instead of Interface. Please Every time you inject somethin - use Interface instead of Object